lib/db/schema/patch-095-094.sql

```diff
@@ -0,0 +1,15 @@
+-- SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+-- DROP PROCEDURE `sessionWithDevice_18`;
+-- DROP PROCEDURE `account_7`;
+-- DROP PROCEDURE `accountRecord_6`;
+-- DROP PROCEDURE `resetAccount_11`;
+-- DROP PROCEDURE `verifyEmail_7`(
+-- DROP PROCEDURE `setPrimaryEmail_5`;
+-- DROP PROCEDURE `deleteEmail_4`;
+-- DROP PROCEDURE `deleteTotpToken_3`;
+-- DROP PROCEDURE `updateTotpToken_3`;
+
+-- XXX TODO: drop columns
```

Comment:
It's kind of a moot point, since @shane-tomlinson suggested we stop doing these reverse migrations anyway, but I just wanted to check you hadn't forgotten about this `TODO` comment.